newIDE/app/src/Utils/GDevelopServices/Game.js

```diff
@@ -63,6 +64,64 @@ export type AllShowcasedGames = {
   filters: Filters,
 };
 
+export const allGameCategories = [
+  'action',
+  'adventure',
+  'shooter',
+  'platformer',
+  'rpg',
+  'horror',
+  'strategy',
+  'puzzle',
+  'story-rich',
+  'survival',
+  'racing',
+  'building',
+  'simulation',
+  'sport',
+  'multiplayer',
+  'leaderboard',
+];
+
+export const getCategoryName = (category: string) => {
```

Comment:
Should this function accept i18n to translate labels?

Reply:
totally right 👍